src/object-marker.scss

```diff
@@ -0,0 +1,83 @@
+@import "./new-settings";
+@import "./mixins";
+
+/*!
+.fd-object-marker
+*/
+$block: #{$fd-namespace}-object-marker;
+
+.#{$block} {
+  // ICON VARIABLES
+  $fd-object-marker-font-size: 1rem !default;
+  $fd-object-marker-color: var(--sapContent_MarkerIconColor) !default;
+  $fd-object-marker-text-color: var(--sapContent_LabelColor) !default;
+  $fd-object-marker-padding-left: 0.5rem !default;
+  $fd-object-marker-padding-right: 0.25rem !default;
+  // TEXT VARIABLES
+  $fd-object-marker-font-family: var(--sapFontFamily) !default;
+
+  @include fd-reset();
+
+  font-family: $fd-object-marker-font-family;
+  font-size: $fd-object-marker-font-size;
+  max-width: 100%;
+  color: $fd-object-marker-text-color;
+  word-break: break-word;
+  display: inline-flex;
+  align-items: center;
+  line-height: 1;
+
+  &::before {
+    color: $fd-object-marker-color;
+    padding: 0 $fd-object-marker-padding-right 0 0;
+  }
+
+  // CLICKABLE OBJECT STATUS
+  &--link {
+    display: inline-block;
+    text-decoration: none;
+    color: $fd-object-marker-color;
+    cursor: pointer;
+
+    &::before {
+      position: relative;
+    }
+
+    &:hover {
+      text-decoration: underline;
+
+      &::before {
+        text-decoration: none;
+      }
+    }
+
+    &:visited {
+      text-decoration: none;
+    }
+  }
+
+  // ICON ONLY MODE
+  @include fd-empty() {
+    justify-content: center;
+    padding-right: 0;
+    padding-left: 0;
+
+    &::before {
+      padding-right: 0;
+      padding-left: $fd-object-marker-padding-left;
```

Comment:
This is not correct. 
<img width="395" alt="Screen Shot 2020-07-07 at 2 42 28 PM" src="https://user-images.githubusercontent.com/39598672/86827534-19c5ac80-c060-11ea-9fbf-a6fa7c06d391.png">

The specs specify that when two Object Markers are next to each other (adjacent siblings) the space between them should be 0.5 rem. As an example is given Icon and Icon but this does not exclude the other combinations. Which means that you should apply this spacing not only in "empty" state of the component. 
In your case this padding is always applied and it shouldn't be the case. It's only when you have 2 elements that are siblings.
Also it should be a margin, not a padding as padding is influencing the width of the element. I suggest you check if you can use the `Adjacent Sibling Selector` and apply margin left to the Object Marker that is following another Object Marker.